### PR TITLE
Update deprecated syntax

### PR DESCRIPTION
**Pull request by musm**
Will merge this soon, since tests pass locally

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,2 @@
 julia 0.6
-Compat 0.9.5
+Compat 0.28.0
```
Comment (tkelman) [inline REQUIRE]: 0.27 https://github.com/JuliaLang/Compat.jl/commit/d0f4b25d2295cf3951a656f702e8920aa1b907a5

Comment (musm) [inline REQUIRE]: I put 0.28 since 0.27 has a weird warning that was fixed in https://github.com/JuliaLang/Compat.jl/pull/383

Comment (tkelman) [inline REQUIRE]: that's a warning, not an error. 0.27 still works.

Comment (musm) [inline REQUIRE]: Is it ok to delete the old tag?

Comment (tkelman) [inline REQUIRE]: as long as you replace it before travis starts running on the open metadata pr, yeah

**Pull request merged**